schemas/target/target-classification.example.1.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "123456:2:0:400001",
+  "xdm:activityName": "My Activity A",
+  "xdm:activityId": "123456",
+  "xdm:experienceName": "My Experience 2",
+  "xdm:experienceId": "2",
+  "xdm:activityExperience": "My Activity A > My Experience 2"
+}
```

Comment:
Because this activity has a 4th value in the id, I would expect that the algorithm fields are also filled out.